cpp/doxygen/footer.html

```diff
@@ -0,0 +1,3 @@
+<script src="https://docs.rapids.ai/assets/js/custom.js"></script>
+<link rel="stylesheet" href="https://docs.rapids.ai/assets/css/custom.css">
+
```

Comment:
Cool. Do these files not need a copyright header?

Reply:
Yup, added now.